cmd/humioctl/filter_alerts.go

```diff
@@ -0,0 +1,34 @@
+// Copyright © 2020 Humio Ltd.
```

Comment:
Should this be updated?
```suggestion
// Copyright © 2024 CrowdStrike
```

Reply:
Yes I believe so. I will update for filter alert. Might change them all in another PR.